documents/customer.md

```diff
@@ -155,6 +155,249 @@ Customer customer = client.Customer.Fetch("cust_M462yViJrhNrQc");
 
 -------------------------------------------------------------------------------------------------------
 
+### Add Bank Account of Customer
+
+```C#
+string customerId = "cust_N5mywh91sXB69O";
+
+Dictionary<string, object> customerRequest = new Dictionary<string, object>();
+customerRequest.Add("ifsc_code", "UTIB0000194");
+customerRequest.Add("account_number", "916010082985661");
+customerRequest.Add("beneficiary_name", "Pratheek");
+customerRequest.Add("beneficiary_address1", "address 1");
+customerRequest.Add("beneficiary_address2", "address 2");
+customerRequest.Add("beneficiary_address3", "address 3");
+customerRequest.Add("beneficiary_address4", "address 4");
+customerRequest.Add("beneficiary_email", "random@email.com");
+customerRequest.Add("beneficiary_mobile", "8762489310");
+customerRequest.Add("beneficiary_city", "Bangalore");
+customerRequest.Add("beneficiary_state", "KA");
+customerRequest.Add("beneficiary_country", "IN");
+
+BankAccount refund = client.Customer.Fetch(customerId).DeleteBankAccount(customerRequest);
```

Comment:
From security perspective, can you confirm if this data is fake?

Reply:
I have removed `beneficiary_mobile` & `account_number `